src/storage/limits.ts

```diff
@@ -115,3 +115,7 @@ export function parseFileSizeToBytes(valueWithUnit: string) {
 export function isUuid(value: string) {
   return /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(value)
 }
+
+export function isEmptyFolder(object: string) {
+  return object.endsWith('.emptyFolderPlaceholder')
```

Comment:
I reckon this gets removed whenever an object is uploaded in the folder

Reply:
not always, is it a client side logic.
However, this PR allow creating this special file even if mime-type are defined